fdbserver/RestoreMaster.actor.cpp

```diff
@@ -243,14 +251,12 @@ ACTOR static Future<Void> loadFilesOnLoaders(Reference<RestoreMasterData> self,
 	    .detail("BeginVersion", versionBatch.beginVersion)
 	    .detail("EndVersion", versionBatch.endVersion);
 
-	Key mutationLogPrefix;
-	std::vector<RestoreFileFR>* files;
+	// Key mutationLogPrefix;
+	std::vector<RestoreFileFR>* files = NULL;
```

Comment:
```suggestion
	std::vector<RestoreFileFR>* files = nullptr;
```
C++ prefers `nullptr`.